http-server-netty/src/main/java/io/micronaut/http/server/netty/RoutingInBoundHandler.java

```diff
@@ -443,6 +443,27 @@ protected void channelRead0(ChannelHandlerContext ctx, io.micronaut.http.HttpReq
         RouteMatch<?> route;
 
         final String requestMethodName = request.getMethodName();
+        MediaType contentType = request.getContentType().orElse(null);
+
+        Optional<Boolean> multipartEnabled = serverConfiguration.getMultipart().getEnabled();
```

Comment:
thing this should be unwrapped from the optional and assigned to a field to avoid the performance overhead for each request